op-program/host/flags/flags.go

```diff
@@ -107,6 +112,7 @@ var programFlags = []cli.Flag{
 	L1TrustRPC,
 	L1RPCProviderKind,
 	Exec,
+	Server,
```

Comment:
Since `exec` and `server` are mutually exclusive, perhaps we can simplify configs by having both under a `--mode` option.

Reply:
hmm, maybe but how do we specify which client executable to run in that case? I guess you could do `--exec <client.exe>` or `--exec server` but it feels a bit awkward and means you can't run a program called "server" as the client.

Reply:
You're right. I assumed `exec` is a boolean flag.